### ISSUE TITLE
ruaml.yaml.load raises warning

### ISSUE THREAD

**Issue by calve** [scrapinghub/dateparser#251]
Hi. It looks like the last version of ruamel.yaml raises a warning when calling `load()`, and suggest using `safe_load` to avoid arbitrary code execution via configuration files. Here is a sample stacktrace from a project of mine importing `dateparser`

```
    /opt/unblock_rule/venv/lib/python3.4/site-packages/dateparser/conf.py:47: UnsafeLoaderWarning: 
    The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
    Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
    Alternatively include the following in your code:
    
      import warnings
      warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)
    
    In most other cases you should consider using 'safe_load(stream)'
      cls._yaml_data = load_yaml(data).pop('settings', {})
    /opt/unblock_rule/venv/lib/python3.4/site-packages/dateparser/languages/loader.py:46: UnsafeLoaderWarning: 
    The default 'Loader' for 'load(stream)' without further arguments can be unsafe.
    Use 'load(stream, Loader=ruamel.yaml.Loader)' explicitly if that is OK.
    Alternatively include the following in your code:
    
      import warnings
      warnings.simplefilter('ignore', ruamel.yaml.error.UnsafeLoaderWarning)
    
    In most other cases you should consider using 'safe_load(stream)'
      data = yaml.load(data)

```

I suggest replacing all occurence of `yaml.load()` by `yaml.safe_load()`

**Comment by nicktacular** [comment]
It would be great to get this merged in. We're seeing this everywhere in our systems.

**Comment by sheriferson** [comment]
I know this issue is closed but I wanted to ask: is there a plan for a PyPI release with a version that has this fixed? As far as I can tell, the latest published version is 0.5.0 and that warning is raised every time dateparser is imported.

**Comment by nicktacular** [comment]
Look like 0.5.1 has been released: https://pypi.python.org/pypi/dateparser
